scripts/components/networking/statssynchronizercomponent/StatsSynchronizerComponent.gd

```diff
@@ -380,24 +383,32 @@ func get_attack_damage() -> float:
 
 
 func apply_boost(boost: Boost):
-	for statName in boost.statBoostDict:
+	#Override any boost with an identical identifier, ignore if this boost doesn't have one.
```

Comment:
Styling for comment is: "#<Space><Text>"

Reply:
I don't quite understand, isn't that how it looks already?

Reply:
#Override any boost with an identical identifier, ignore if this boost doesn't have one.

Should be:

# Override any boost with an identical identifier, ignore if this boost doesn't have one.